dev-libs/beecrypt/beecrypt-4.2.1.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="A strong and fast cryptography toolkit"
+DESCRIPTION="BeeCrypt is an ongoing project to provide a strong and fast \
+cryptography toolkit. It includes entropy sources, random generators, block \
+ciphers, hash functions, message authentication codes, multiprecision integer \
+routines, and public key primitives."
+HOMEPAGE="http://beecrypt.sourceforge.net"
+COPYRIGHT="2009 Bob Deblier"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/beecrypt/beecrypt-4.2.1.tar.gz"
```

Comment:
Please change this to
```
SOURCE_URI="https://downloads.sourceforge.net/beecrypt/beecrypt-$portVersion.tar.gz"
```
(Well,  I know this project did not get new releases for a long time, but it could some day ;-)